src/Calculator/Resources/de-DE/Resources.resw

```diff
@@ -847,7 +847,7 @@
   </data>
   <data name="equalButton.[using:Windows.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
     <value>Gleich</value>
-    <comment>Screen reader prompt for the invert button on the scientific operator keypad</comment>
+    <comment>Screen reader prompt for the equal button on the scientific operator keypad</comment>
```

Comment:
This is a good catch.  Can you make _this_ change to the en-us resource?  The comment will then get fixed across all of the resources the next time it round-trips through our localization process.